Makefile

```diff
@@ -336,8 +335,8 @@ out/doc/api/%.html: doc/api/%.md
 		fi
 	[ -x $(NODE) ] && $(NODE) $(gen-html) || node $(gen-html)
 
-docopen: out/doc/api/all.html
-	-google-chrome out/doc/api/all.html
+docopen: $(apidocs_html)
```

Comment:
@silverwind Actually `doc-only` is the odd man here I guess. Because we have `docclean` as well :D